### PR TITLE
Code cleanup

### PR DESCRIPTION
**Pull request by mickel1138**

### CODE REVIEW

**Review by fightingtexasaggie — COMMENTED** [review]
In particular, with strings on mixed types, ints or other non-class types, why not keep the description? There's no other place to put it.  I totally agree on removing the @params for any class though, since it's best described in the class itself.

File: src/Controller/Admin/DigitalResourceTypeDescriptorCrudController.php [hunk 1]
```diff
@@ -72,10 +62,6 @@ public function configureCrud(Crud $crud): Crud
 
     /**
      * Configure fields for CRUD.
-     *
-     * @param string $pageName
```
Comment (fightingtexasaggie) [inline src/Controller/Admin/DigitalResourceTypeDescriptorCrudController.php:76]: keep?

File: src/Controller/Admin/DigitalResourceTypeDescriptorCrudController.php [hunk 2]
```diff
@@ -89,8 +75,6 @@ public function configureFields(string $pageName): iterable
     /**
      * Overwrite for when entity is created.
      *
-     * @param string $entityFqcn
```
Comment (fightingtexasaggie) [inline src/Controller/Admin/DigitalResourceTypeDescriptorCrudController.php:92]: keep?

File: src/Controller/UI/DatasetReviewController.php [hunk 3]
```diff
@@ -177,31 +165,28 @@ protected function eligibiltyForReview(string $udi, Request $request)
...
     /**
      * Gets Latest dataset submission and checks for errors to add in the flash bag.
-     *
-     * @param Request $request The Symfony request object.
-     * @param Dataset $dataset A dataset instance..
-     *
-     * @return DatasetSubmission  A dataset submission instance.
+
      */
     private function latestDatasetSubmissionForReview(Request $request, Dataset $dataset)
```
Comment (fightingtexasaggie) [inline src/Controller/UI/DatasetReviewController.php:176]: missing return type

File: src/Controller/UI/DatasetReviewController.php [hunk 4]
```diff
@@ -233,10 +218,9 @@ private function latestDatasetSubmissionForReview(Request $request, Dataset $dat
...
      *
-     * @param Request $request          The Symfony request object.
-     * @param string  $udi              The UDI entered by the user.
-     * @param string  $noticeCode       The type of Notice/Error generated, non-numeric.
-     * @param string  $reviewerUserName Reviewer Username for the Dataset submission review.
+     * @param string  $udi              the UDI entered by the user
+     * @param string  $noticeCode       the type of Notice/Error generated, non-numeric
+     * @param string  $reviewerUserName reviewer Username for the Dataset submission review
      *
      * @return void
```
Comment (fightingtexasaggie) [inline src/Controller/UI/DatasetReviewController.php:225]: missing return type below

File: src/Controller/UI/DatasetReviewController.php [hunk 5]
```diff
@@ -270,11 +254,8 @@ private function addToFlashDisplayQueue(Request $request, string $udi, string $n
...
      *
-     * @param string            $udi               The UDI entered by the user.
-     * @param Dataset           $dataset           The Dataset.
-     * @param DatasetSubmission $datasetSubmission The Dataset Submission.
-     *
-     * @return Response A Response instance.
+     * @param string            $udi               the UDI entered by the user
+
      */
     protected function makeSubmissionForm(string $udi, Dataset $dataset = null, DatasetSubmission $datasetSubmission = null)
```
Comment (fightingtexasaggie) [inline src/Controller/UI/DatasetReviewController.php:260]: make private? also, missing return type

File: src/Controller/UI/DatasetReviewController.php [hunk 6]
```diff
@@ -414,7 +395,7 @@ private function createNewDatasetSubmission(DatasetSubmission $datasetSubmission
     /**
      * Create an entity for each new review.
      *
-     * @param Entity $entity A DatasetSubmission or DatasetSubmissionReview to base this DatasetSubmission on.
+     * @param Entity $entity a DatasetSubmission or DatasetSubmissionReview to base this DatasetSubmission on
```
Comment (fightingtexasaggie) [inline src/Controller/UI/DatasetReviewController.php:398]: Do we need this?

File: src/Controller/UI/DatasetReviewController.php [hunk 7]
```diff
@@ -531,22 +510,22 @@ public function postAction(Request $request, EntityManagerInterface $entityManag
...
 
     /**
-     * To check the if files are being processed
+     * To check the if files are being processed.
      *
-     * @param DatasetSubmission $datasetSubmission A dataset submission instance.
+     * @param DatasetSubmission $datasetSubmission a dataset submission instance
      *
-     * @return boolean
+     * @return bool
```
Comment (fightingtexasaggie) [inline src/Controller/UI/DatasetReviewController.php:528]: move to signature?

File: src/Controller/UI/SearchPageController.php [hunk 8]
```diff
@@ -64,17 +61,15 @@ public function defaultAction()
...
-     *
-     * @param Request $request    The Symfony request object.
-     * @param Search  $searchUtil Search utility class object.
+
      *
      * @Route("/search/results", name="pelagos_app_ui_searchpage_results")
      *
      * @return Response
      */
     public function getSearchResults(Request $request, Search $searchUtil)
```
Comment (fightingtexasaggie) [inline src/Controller/UI/SearchPageController.php:70]: move return to signature

File: src/Entity/DIF.php [hunk 9]
```diff
@@ -412,7 +412,7 @@ class DIF extends Entity
      *
      * Initializes status to unsubmitted.
      *
-     * @param Dataset $dataset The dataset this DIF identifies.
+     * @param Dataset $dataset the dataset this DIF identifies
      */
     public function __construct(Dataset $dataset = null)
```
Comment (fightingtexasaggie) [inline src/Entity/DIF.php:417]: return type

File: src/Controller/Admin/DigitalResourceTypeDescriptorCrudController.php (hunk 1 above)
Comment (mickel1138) [reply src/Controller/Admin/DigitalResourceTypeDescriptorCrudController.php:76]: Yes, if there was a good description that goes with it,

File: src/Controller/UI/DatasetReviewController.php (hunk 3 above)
Comment (mickel1138) [reply src/Controller/UI/DatasetReviewController.php:176]: Yes, but decided not to do these in this go around.

File: src/Controller/UI/DatasetReviewController.php (hunk 6 above)
Comment (mickel1138) [reply src/Controller/UI/DatasetReviewController.php:398]: Nope

File: src/Controller/UI/DatasetReviewController.php (hunk 5 above)
Comment (mickel1138) [reply src/Controller/UI/DatasetReviewController.php:260]: Yes can, but didn't do this. But in the future you should do that.

File: src/Controller/UI/DatasetReviewController.php (hunk 7 above)
Comment (mickel1138) [reply src/Controller/UI/DatasetReviewController.php:528]: Confused? You mean return type?

**Review by fightingtexasaggie — APPROVED** [review]
solid start

**Pull request merged**